name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/ComdirectPDFExtractor.java

```diff
@@ -76,7 +73,7 @@
 @SuppressWarnings("nls")
 public class ComdirectPDFExtractor extends AbstractPDFExtractor
 {
-    private static record SaleTaxPair(Item sale, Item tax)
+    private static record TransactionPair(Item transaction, Item tax)
```

Comment:
There is already a class named [TransactionPair](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio/src/name/abuchen/portfolio/model/TransactionPair.java). Therefore I suggest to name this one differently to avoid confusion.

Reply:
done... renamed